p2p-specs/p2p-interface.md

```diff
@@ -494,7 +499,29 @@ The response MUST be encoded as an SSZ-container.
 
 The response MUST consist of a single `response_chunk`.
 
-##### `PooledUserOpsByHashes`
+#### PooledUserOpHashes
+
+**Protocol ID:** `/account_abstraction/erc4337/req/pooled_user_op_hashes/1/`
+
+Request Content:
+
+```
+(
+  mempool: Bytes32
+)
+```
+
+Response Content: 
+```
+(
+  List[Bytes32, MAX_OPS_PER_REQUEST]
+)
+```
+
+The `pooled_user_op_hashes` requests UserOps from the recipients UserOp mempool for a given mempool_id. The recommended soft limit for PooledUserOpHashes requests is MAX_OPS_PER_REQUEST hashes. The recipient may enforce an arbitrary limit on the response (size or serving time), which must not be considered a protocol violation. This message is requested UserOp mempool of all connected peers as soon as bundler node startups.
+
+
+#### PooledUserOpsByHashes
```

Comment:
Small nit: I'd remove the s at the end and call this `PooledUserOpsByHash` similar to the consensus layer's [BeaconBlocksByRoot](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#beaconblocksbyroot)